geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/DescribeJndiBindingCommand.java

```diff
@@ -40,6 +40,7 @@
   private static final String DESCRIBE_JNDIBINDING__HELP =
       "Describe the configuration of the given jndi binding.";
   private static final Function LIST_BINDING_FUNCTION = new ListJndiBindingFunction();
+  public static final String JNDI_PROPERTIES_SECTION = "jndi-properties";
```

Comment:
This is probably the most nit-picky nit I've ever picked, which is a lot coming from me...

But I think a `public static`s should be the first fields declared in a class.